io500-info-editor.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+import sys
+import json
+
+# This tool allows to view and edit the options in an existing schema file.
+# In order to get started download the two files:
+# wget https://www.vi4io.org/lib/plugins/newcdcl/scripts/schema-io500.json
+# wget https://www.vi4io.org/lib/plugins/newcdcl/scripts/site-io500.json
```

Comment:
Does it make sense to include these files in the commit to the io500 repo, so they are immediately available for use?  Alternately, having the script be able to fetch them would be very useful.